### ISSUE TITLE
positive = logits.slice_axis(axis=2, begin=1, end=-1)

### ISSUE THREAD

**Issue by mifan0208** [dmlc/gluon-cv#740]
I use OHEM to run ssd,but the positive = logits.slice_axis(axis=2, begin=1, end=-1) can't get the last category。When my category's num is 1,this is wrong,please help me.

**Comment by zhreshold** [comment]
try begin=1, end=None?

**Comment by mifan0208** [comment]
> try begin=1, end=None?
I tried this.The result is OK。The source code has a problem.
